cmd/unifi_exporter/main.go

```diff
@@ -9,49 +9,70 @@ import (
 	"net/http"
 	"strings"
 	"time"
+	"io/ioutil"
 
+	"gopkg.in/yaml.v2"
 	"github.com/mdlayher/unifi"
 	"github.com/mdlayher/unifi_exporter"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+type Config struct {
+	TelemetryAddr string `yaml: "telemetryaddr"`
```

Comment:
I'd rather call this `listen` if both host and port are specified or split it into `listen.address` and `listen.port`. I like the second option better as you can stick to the default port while binding to a different IP without having to specify both of them. I think `listen` is more intuitive as the point where a daemon listens for connections, rather than `telemetry` (seems to be more common as well).

Reply:
To be fair, you make good points, but I didn't want to step on the original author's toes too much by unilaterally deciding that his variable names were wrong and mine were right. That said, in the original readme it does say it was adapted from another, so he may have just left those the way they were and weren't personal choices of his. I'll give it some thought.

Reply:
Yeah I am really rude when it comes to such things :D. Maybe @mdlayher could comment what he thinks of this or why he chose the current version in the first place?